src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -222,7 +222,7 @@ internal Silo(SiloInitializationParameters initializationParams)
             var services = new ServiceCollection();
             services.AddSingleton(this);
             services.AddSingleton(initializationParams);
```

Comment:
maybe we should remove this line and only have the one registration:
`services.AddSingleton<ILocalSiloDetails>(initializationParams);`